app/Resources/views/pagelayout.html.twig

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en-US"
+      class="no-js">
+    <head>
+        {% if content is defined and title is not defined %}
+            {% set title = ez_content_name(content) %}
+        {% endif %}
+        <title>{{ title|default('Home')|trans }} - {{ 'eZ Platform' }}</title>
+
+        <meta http-equiv="Content-Type"
+              content="text/html; charset=utf-8" />
```

Comment:
I am aware that the ending slash is optional in HTML5. However, it is not wrong to use them, and it offers a few benefits which we may discuss if desired. Reference:

https://www.w3.org/TR/html5/syntax.html#start-tags (Item number 6):

>Then, if the element is one of the void elements, or if the element is a foreign element, then there may be a single "/" (U+002F) character. This character has no effect on void elements, but on foreign elements it marks the start tag as self-closing.

http://www.standardista.com/6-best-practices-for-marking-up-your-html5/ (Item number 3.)